### DISCUSSION TITLE
A smaller result view that displays the result over the input

### DISCUSSION THREAD

**Discussion by ehrenschwan-gh** [monkeytypegame/monkeytype#4222] [Ideas]
I would really like the **option** to have a smaller reduced result view that sits above the input. So that I'm not directed to the result page and have to restart the test. It would also make auto failing be feasible for me. `tab` for restarting is already pretty nice but it's still a key  to hit to restart. If the results were displayed in a reduced view above the input it would make it so that one could basically always be typing.

**Comment by aviiciii** [comment]
Yes, that seems like a good idea!
